samples/gym-bipedal-walker-sample/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# this is one of the cached base images available for ACI
+FROM python:3.7.4
+
+# Install libraries and dependencies
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    build-essential \
+    cmake \
+    zlib1g-dev \
+    swig
+
+# Install dependencies
+RUN pip3 install -U setuptools \
+&& pip3 install git+https://github.com/microsoft/bonsai-common \
```

Comment:
why not put both of these in `requirements.txt` like the other samples?

Reply:
(for some reason, it wasn't installing properly and i'd get conflicting version issues for gym. 
```
ERROR: Cannot install -r requirements.txt (line 6), gym[atari,box2d,classic-control]==0.10.11, gym[atari,box2d,classic-control]==0.10.8, gym[atari,box2d,classic-control]==0.10.9, gym[atari,box2d,classic-control]==0.11.0, gym[atari,box2d,classic-control]==0.12.0, gym[atari,box2d,classic-control]==0.12.1, gym[atari,box2d,classic-control]==0.12.4, gym[atari,box2d,classic-control]==0.12.5, gym[atari,box2d,classic-control]==0.12.6, gym[atari,box2d,classic-control]==0.13.0, gym[atari,box2d,classic-control]==0.13.1, gym[atari,box2d,classic-control]==0.14.0, gym[atari,box2d,classic-control]==0.15.3, gym[atari,box2d,classic-control]==0.15.4, gym[atari,box2d,classic-control]==0.15.6, gym[atari,box2d,classic-control]==0.15.7, gym[atari,box2d,classic-control]==0.16.0, gym[atari,box2d,classic-control]==0.17.0, gym[atari,box2d,classic-control]==0.17.1, gym[atari,box2d,classic-control]==0.17.2, gym[atari,box2d,classic-control]==0.17.3, gym[atari,box2d,classic-control]==0.18.0 and gym[atari,box2d,classic-control]==0.18.3 because these package versions have conflicting dependencies.
ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies